web/src/utils/date.ts

```diff
@@ -1,6 +1,6 @@
 import moment from 'moment'
 
-import { PST_UTC_OFFSET, PDT_UTC_OFFSET } from 'utils/constants'
+import { PST_UTC_OFFSET, PDT_UTC_OFFSET } from './constants'
```

Comment:
I thought we were always using absolute paths for imports?

Reply:
Yeah... but when Cypress references this file, it doesn't like absolute paths